### ISSUE TITLE
Can't run OAuth instance

### ISSUE THREAD

**Issue by quentindelattre** [oauth-io/oauthd#175]
I encounter problems when I try to run the OAuth instance in my directory.

I followed all the instructions, doubled check everything and even tried a fresh re-install of node, redis and oauthd. Yet, every time I run the `oauthd start` command, I get this output:

```
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
```

Do you have any suggestion on how to fix this ?

Thanks in advance !
^Q

**Comment by iamlogiq** [comment]
Same problem +1

**Comment by rinoto** [comment]
Same Problem here using Mac 10.10.5

**Comment by cwdcwd** [comment]
[This](https://github.com/trentm/node-bunyan/issues/216#issuecomment-186281875) seems to work when trying to install the server but then the plugins don't seem to work. Same error:
```
$ node ./bin/cli/index start
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
```

**Comment by bumpmann** [comment]
I think you are using a too recent node version. Some deps are still stuck to 3.x

**Comment by cwdcwd** [comment]
what's the recommended version for node then?

**Comment by bumpmann** [comment]
i m still not sure, but using https://github.com/tj/n you can set node to version 0.12.x which is maintained with the security updates

**Comment by thyb** [comment]
We've just released the version 1.0.0-rc.1 which upgrade all dependencies. You can now use oauthd with the latest Node version (currently v7) without getting the error described in this issue.
